ecosystem/indexer-server/typescript/src/services/service_v1.test.ts

```diff
@@ -205,19 +224,53 @@ test('/ownerships/all', async () => {
   } = ownership2;
   const fetchedOwnerships = await axios.get<ownerships[]>(`http://localhost:${URL_PORT}/${version}/ownerships/all`);
   expect(fetchedOwnerships.data.length).toBeGreaterThan(1);
-  await cleanupOwnership(owner1, token_id1);
-  await cleanupOwnership(owner2, token_id2);
+  await cleanupOwnership(owner1!, token_id1!);
```

Comment:
What's the postfix for?

Reply:
as nonnull.

Reply:
It should throw an error anyways if its null though right?

Reply:
Lint is yelling at me lol.